src/libraries/NetCoreAppLibrary.props

```diff
@@ -1,7 +1,33 @@
 <Project>
   <PropertyGroup>
     <!-- The trailing semicolon is important for each entry! -->
+    <NetFxReference>
+      mscorlib;
+      Microsoft.VisualBasic;
+      System;
+      System.ComponentModel.DataAnnotations;
+      System.Configuration;
+      System.Core;
+      System.Data;
+      System.Drawing;
+      System.IO.Compression.FileSystem;
+      System.Net;
+      System.Numerics;
+      System.Runtime.Serialization;
+      System.Security;
+      System.ServiceProcess;
+      System.ServiceModel.Web;
+      System.Transactions;
+      System.Web;
+      System.Windows;
+      System.Xml;
+      System.Xml.Serialization;
+      System.Xml.Linq;
+      WindowsBase;
+    </NetFxReference>
     <NetCoreAppLibrary>
+      $(NetFxReference)
```

Comment:
Why do we now need these to appear in the `NetCoreAppLibrary` property?

Reply:
NetCoreAppLibrary is the list of libraries which compose the shared framework which the .NETFramework shims are part of.